lib/util/normalize.js

```diff
@@ -28,7 +28,7 @@ export const normalizePost = (post, buckets, final) => {
   if (!post) return
   const { groups, people } = buckets
   convertList(groups, post, 'group')
-  convertList(people, post, 'voter')
+  convertList(people, post, 'voter') // TODO: not really sure what this voter stuff is doing
```

Comment:
i see it being used in evo in getMemberVotes, maybe you are not using it anymore, but maybe still being used in mobile and we should flag to remove?